### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ChrisAntaki**
NextBus supports JSON actually. Just replace "publicXMLFeed" with "publicJSONFeed". For example:
http://webservices.nextbus.com/service/publicJSONFeed?command=vehicleLocations&a=sf-muni&r=1&t=1436558856779

### CODE REVIEW

**Comment by morganney** [comment]
It is true that NextBus serves JSON from that endpoint, however it is not something they endorse or share publicly.  It could be removed at any point without warning so I would not recommend using it for a production application. Moreover, it converts everything to a string in order to prevent lost precision on the lat/lon attributes of the XML feed, so your application would have to do lots of typecasting.

In short, it was an afterthought of NextBus in an attempt to serve much desired JSON that never really panned out.

**Comment by ChrisAntaki** [comment]
Interesting! I would guess that they'd be moving towards releasing the JSON API publicly and endorsing it. Taking it away would be a step backwards, and potentially break a lot of apps.

In my experience, the JSON API has worked perfectly so far. There's one JSON serialization bug where an array with one object is returned as the object itself. Using Node.js, I've been writing `[].concat(anArrayPerhaps)` to get around that issue. Casting the latitude and longitude to numbers hasn't been challenging either, at least not yet :wink:.

It almost seems like they just need a final pass of QA. Maybe they just never allocated the funding because the agencies themselves were never asking for the feature?

**Comment by morganney** [comment]
When I was still working there we developed a JSON API which is identical to restbus (except the `_links` property). This is what they are currently using internally to build their applications. They just haven't released it to the public yet which is part of the reason I wrote restbus.

If you visit their homepage and use dev tools you can see it in action.

**Comment by ChrisAntaki** [comment]
Thanks for the heads up!
